src/ssl.c

```diff
@@ -36426,7 +36490,89 @@ WOLFSSL_STACK* wolfSSL_PKCS7_get0_signers(PKCS7* pkcs7, WOLFSSL_STACK* certs,
 
     return signers;
 }
-#endif
+
+/******************************************************************************
+* wolfSSL_PEM_write_bio_PKCS7 - writes the PKCS7 data to BIO
+*
+* RETURNS:
+* returns WOLFSSL_SUCCESS on success, otherwise returns WOLFSSL_FAILURE
+*/
+
+int wolfSSL_PEM_write_bio_PKCS7(WOLFSSL_BIO* bio, PKCS7* p7)
+{
+    WOLFSSL_ENTER("wolfSSL_PEM_write_bio_PKCS7()");
```

Comment:
WOLFSSL_ENTER should be moved down to after declaration of local variables. Windows builds and older compilers will error out on operations performed before variable declaration